scripts/devenv-builder/create-vm.sh

```diff
@@ -63,6 +63,7 @@ virt-install \
     --location ${ISOFILE} \
     --initrd-inject=${KICKSTART_FILE} \
     --extra-args \"inst.ks=file:/$(basename "${KICKSTART_FILE}")\" \
+    --graphics vnc,listen=0.0.0.0 \
```

Comment:
How do we connect to this VM using VNC? Should add RPMs to some configure script (i.e. `manage-vm.sh config`, etc.)?

Reply:
This enables the VNC console in cockpit, so you don't need any other software.